schemas/3DScenesConfiguration/v1.0.0/3DScenesConfiguration.schema.json

```diff
@@ -236,6 +236,62 @@
                 "time"
             ]
         },
+        "IDataHistoryBasicTimeSeries": {
+            "type": "object",
+            "description": "A basic timeseries to be rendered in the chart of the data history widget",
+            "additionalProperties": false,
+            "properties": {
+                "expression": {
+                    "type": "string"
+                },
+                "unit": {
+                    "type": "string"
+                },
+                "label": {
+                    "type": "string"
+                }
+            },
+            "required": ["expression"]
+        },
+        "IDataHistoryTimeSeries": {
+            "type": "array",
+            "description": "A list of timeseries to render in the chart",
+            "items": {
+                "$ref": "#/$defs/IDataHistoryBasicTimeSeries"
+            }
+        },
+        "IDataHistoryConnectionString": {
+            "type": "string",
+            "description": "Timeseries database connection string in key1=value1;key2=value2;key3=value3 format that is used for the connection of a data history widget"
+        },
+        "IDataHistoryChartYAxisType": {
+            "type": "string",
+            "enum": ["Shared", "Independent"]
```

Comment:
since I use the generated types directly in the UI as dropdown options I wanted to keep them like this: 
![image](https://user-images.githubusercontent.com/45217314/195207094-29c68a90-751a-465a-997a-0696df71bdc2.png)
![image](https://user-images.githubusercontent.com/45217314/195207314-81331bb9-6355-4517-ae0a-2f999fcdbfd4.png)

I can do the casing manipulation on the UI side though

Reply:
nevermind, looks like I was already using translation for it. Okay making the type all lowercase